site/content/docs/main/resource-filtering.md

```diff
@@ -201,3 +203,121 @@ Kubernetes namespace resources to exclude from the backup, formatted as resource
+
+   Users need to create one configmap in Velero install namespace from a YAML file that defined resource policies. The creating command would be like the below:
+   ```bash
+   kubectl create cm <configmap-name> --from-file <yaml-file> -n velero
+   ```
+2. Creating a backup reference to the defined resource policies
+
+   Users create a backup with the flag `--resource-policies-configmap`, which will reference the current backup to the defined resource policies. The creating command would be like the below:
+   ```bash
+   velero backup create --resource-policies-configmap <configmap-name>
+   ```
+   This flag could also be combined with the other include and exclude filters above
+
+**YAML template**
+
+Velero only support volume resource policies currently, other kinds of resource policies could be extended in the future. The policies YAML config file would look like this:
+- Yaml template:
+    ```yaml
+    # currently only supports v1 version
+    version: v1
+    volumePolicies:
+    # each policy consists of a list of conditions and an action
+    # we could have lots of policies, but if the resource matched the first policy, the latters will be ignored
+    # each key in the object is one condition, and one policy will apply to resources that meet ALL conditions
+    # NOTE: capacity or storageClass is suited for [Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes), and pod [Volume](https://kubernetes.io/docs/concepts/storage/volumes) not support it.
+    - conditions:
+        # capacity condition matches the volumes whose capacity falls into the range
+        capacity: "10,100Gi"
+        # pv matches specific csi driver
+        csi:
+          driver: aws.ebs.csi.driver
+        # pv matches one of the storage class list
+        storageClass:
+        - gp2
+        - standard
+      action:
+        type: skip
+    - conditions:
+        capacity: "0,100Gi"
+        # nfs volume source with specific server and path (nfs could be empty or only config server or path)
+        nfs:
+          server: 192.168.200.90
+          path: /mnt/data
+      action:
+        type: skip
+    - conditions:
+        nfs:
+          server: 192.168.200.90
+      action:
+        type: skip
+    - conditions:
+        # nfs could be empty which matches any nfs volume source
+        nfs: {}
+      action:
+        type: skip
+    - conditions:
+        # csi could be empty which matches any csi volume source
+        csi: {}
+      action:
+        type: skip
+    ```
+
+**Supported conditions**
+
+Currently, Velero supports the volume attributes listed below:
+- capacity: matching volumes have the capacity that falls within this `capacity` range. The capacity value should include the lower value and upper value concatenated by commas. And it has several combinations below:
+  - "0,5Gi" or "0Gi,5Gi" which means capacity or size matches from 0 to 5Gi, including value 0 and value 5Gi
```

Comment:
Not sure about the "0,5Gi" setting.
It looks like "0Gi,5Gi" is more reasonable.
In my testing, "10,100Gi" is not the same as "10Gi,100Gi".

Reply:
Maybe I should make a clearer explanation in the doc, the Unit of the capacity could be `Gi`, `Mi`, `Ki` etc.  
The Unit of the Capacity is the standard K8S storage unit. so "10,100Gi" is not equal with "10Gi,100Gi"

Reply:
updated